### PR TITLE
feat: [OeX_Cred-766] modal app links and qr copy link

### PR DESCRIPTION
**Pull request by kyrylo-kh**

### CODE REVIEW

File: src/components/ProgramCertificateModal/ProgramCertificateModal.jsx [hunk 1]
```diff
@@ -62,6 +66,16 @@ function ProgramCertificateModal({
                   alt={intl.formatMessage(messages.certificateModalQrCodeLabel)}
                 />
               </div>
+              <button
+                className="justify-content-center align-items-center  mt-2 copy-link"
```
Comment (ihor-romaniuk) [inline src/components/ProgramCertificateModal/ProgramCertificateModal.jsx]: You have an extra space before `mt-2` class name

File: src/components/ProgramCertificateModal/ProgramCertificateModal.scss [hunk 2]
```diff
@@ -0,0 +1,36 @@
+.buttons-wrapper {
+    margin-top: 25px;
+    img {
```
Comment (ihor-romaniuk) [inline src/components/ProgramCertificateModal/ProgramCertificateModal.scss:4]: Add an empty line before `img {`

File: src/components/ProgramCertificateModal/ProgramCertificateModal.scss [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+    font-size: 14px;
+    color: #152F52;
+    cursor: pointer;
+    background: none;
+    border: none;
+    margin: 0;
+    padding: 0;
+
+    &:hover {
+        color: black;
```
Comment (ihor-romaniuk) [inline src/components/ProgramCertificateModal/ProgramCertificateModal.scss]: It's better to use `#000`

File: src/components/ProgramCertificateModal/ProgramCertificateModal.scss [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+
+    svg {
+        width: 20px;
+        height: 20px;
+        margin-right: 7px;
+    }
+}
+
+.mobile-button {
+    background-color: #000000;
```
Comment (ihor-romaniuk) [inline src/components/ProgramCertificateModal/ProgramCertificateModal.scss]: It's better to use `#000` (color in a short form)

Comment (ihor-romaniuk) [reply src/components/ProgramCertificateModal/ProgramCertificateModal.scss]: And do you have any rules for hover?

Comment (kyrylo-kh) [reply src/components/ProgramCertificateModal/ProgramCertificateModal.scss]: No, there are static images

**Pull request merged**